app/views/projects/show.html.erb

```diff
@@ -1,5 +1,7 @@
 <h1><%= @project.name %></h1>
 
+<%= render "shared/flash_messages" %>
```

Comment:
You already show the partial in application layout, so remove this or the flash message will show twice when a new project is created.